tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -410,72 +419,97 @@ module.exports = {
      * This does not works in cases where the path starts with a space (eg. "  ParentFolder/file.txt")
      *
      * @param {string} path
-     */
-    waitForFileWithPathVisible: function (path) {
-      const rowSelector = this.getFileRowSelectorByFileName(path)
-      const linkSelector = this.getFileLinkSelectorByFileName(path)
-      return this
-        .useXpath()
-        .waitForElementVisible(rowSelector)
-        .getAttribute(linkSelector, 'innerText', function (result) {
-          this.assert.strictEqual(result.value.trim(), path, 'displayed file name not as expected')
+     * @param {string} elementType
+     */
+    waitForFileWithPathVisible: function (path, elementType = 'file') {
+      if (elementType === 'file') {
+        const rowSelector = this.getFileRowSelectorByFileName(path, elementType)
+        const linkSelector = this.getFileLinkSelectorByFileName(path, elementType)
+        return this
+          .useXpath()
+          .waitForElementVisible(rowSelector)
+          .getAttribute(linkSelector, 'innerText', function (result) {
+            this.assert.strictEqual(result.value.trim(), path, 'displayed file name not as expected')
```

Comment:
why is the waiting and the name checking only done for files and not folders?

Reply:
fixed